book/src/reference/config.md

```diff
@@ -256,6 +256,25 @@ For instance for packages that are a library and a CLI binary, some developers p
 If you use this you *probably* want to set it on `[package.metadata.dist]` and
 not `[workspace.metadata.dist]`. See ["inferring precise-builds"](#inferring-precise-builds) for details.
 
+### hosting
+
+> since 0.5.0
+
+Example: `hosting = ["axodotdev", "github"]
+
+Possible values:
+
+* `axodotdev`: Use Axo Releases (currently in closed beta)
+* `github`: Use Github Releases (default if ci = "github")
```

Comment:
Probably doesn't matter, but the canonical spelling is "GitHub". (No one gets this right)

Reply:
we Extremely Consistently get it wrong throughout all of cargo-dist lmao (or at least, i do)

(literally my mind was blown this year when i noticed it was GitHub like wtf)